boards/FSM/Src/main.c

```diff
@@ -173,19 +175,23 @@ int main(void)
     Io_SharedHardFaultHandler_Init();
 
     Io_FlowMeters_Init(&htim4);
-    primary_flow_meter = App_FlowMeter_Create(Io_FlowMeters_GetPrimaryFlowRate);
-    secondary_flow_meter =
-        App_FlowMeter_Create(Io_FlowMeters_GetSecondaryFlowRate);
+    primary_flow_meter =
+        App_InRangeCheck_Create(Io_FlowMeters_GetPrimaryFlowRate, 1.0f, 30.0f);
+    secondary_flow_meter = App_InRangeCheck_Create(
+        Io_FlowMeters_GetSecondaryFlowRate, 1.0f, 30.0f);
 
     Io_WheelSpeedSensors_Init(&htim16, &htim17);
+    left_wheel_speed_sensor = App_InRangeCheck_Create(
```

Comment:
Why is minimum speed 0.1? Shouldn't it be 0?

Reply:
I went with 0.1 b/c represents the lowest measurable value for the wheel speed.

Reply:
We should have a comment explaining this then